notebooks/query-language-object-based-dialect/01.01.quantifier_capabilities.ipynb

```diff
@@ -0,0 +1,842 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b0f5e933",
+   "metadata": {},
+   "source": [
+    "# Hatchet Query Language\n",
+    "\n",
+    "\n",
+    "This notebook explores [Hatchet](https://github.com/LLNL/hatchet) queries from it's [**Object-based Dialect**](https://hatchet.readthedocs.io/en/latest/query_lang.html) within **Category 1: Quantifier Capabilities**. Below, Fig. 1 provides an overview of all the categories within the Hatcher Query Language.   "
```

Comment:
Overall I think this introduction is fine, but since you state within the table "Category ID". I would also keep this scheme within the sentence. So for instance "This notebook explores Hatchet queries from its Object-based Dialect within Category ID 1: Quantifier Capabilities".